tests/helix/send-to-helix-inner.proj

```diff
@@ -49,35 +49,36 @@
     <_TestRunCommandArguments Include="full" />
   </ItemGroup>
 
-  <Import Project="$(MSBuildThisFileDirectory)send-to-helix-workload-tests.targets" />
+  <Import Project="$(MSBuildThisFileDirectory)send-to-helix-basic-tests.targets" Condition="'$(TestCategory)' == 'basictests'" />
+  <Import Project="$(MSBuildThisFileDirectory)send-to-helix-workload-tests.targets" Condition="'$(TestCategory)' == 'EndToEndTests'" />
```

Comment:
Can we make the .targets file the same as the `$(TestCategory)` and then this could just be:

```
<Import Project="$(MSBuildThisFileDirectory)send-to-helix-$(TestCategory).targets" />
```

Reply:
Hm I'll address this one in the workloads PR. The reason being that the workload tests targets file is being used here for `EndToEndTests`, but it will have some sharing with the workload tests category also in the upcoming PR.